utils.js

```diff
@@ -488,3 +488,24 @@ export const getOptionsFromHash = (hash, label = 'name') => {
 
   return results
 }
+
+export const getPrice = (() => {
+  function max(p) {
+    if (typeof p === 'string' || typeof p === 'number') {
+      return p
+    }
+    const price = `${(p?.maxAmount || p?.amount) + ' ' + (p?.currency || ' ')}`
```

Comment:
where is `amount` defined?

Reply:
Thanks for nice catch:) That was defined by myself in previous commits and stayed in my mind as a fundamental. But for sure it needless and a mistake to let it here. I removed it now.
Already created events  has a **_price_** property which is a string. With  new code **_price_** property becomes an object containing **_{ max:22.00, min:11.00, currency :'$'}_** . **_getPrice_** utils are now capable to handle both situations